netty/src/main/java/io/grpc/transport/netty/GrpcSslContexts.java

```diff
@@ -43,6 +47,16 @@
 public class GrpcSslContexts {
   private GrpcSslContexts() {}
 
+  private static ApplicationProtocolConfig DEFAULT_APN = new ApplicationProtocolConfig(
+          Protocol.ALPN,
+          SelectorFailureBehavior.FATAL_ALERT,
```

Comment:
Acknowledged .... I'll follow up with the Netty guys to see what it will take to get OpenSsl to support our behaviors.